.github/workflows/build.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - run: bun install
       - run: bun run check
       - run: bun run build
+        env:
+          BLOG_BASE_URL: "https://samestep.com"
```

Comment:
I think I'll just hardcode this as a `const` somewhere in `common.tsx`; that way it'll still work with my new `nix build` setup that I added in #4 and #5.